testCourse/questions/orderBlocks/question.html

```diff
@@ -98,3 +98,20 @@
   <pl-answer ranking="1" indent="-1" correct="true">first</pl-answer>
   <pl-answer ranking="2" indent="-1" correct="true">second</pl-answer>
 </pl-order-blocks>
+
+<pl-order-blocks
+answers-name="test-ordered-grading-method-all-or-nothing"
+grading-method="ordered">
+<pl-answer correct="true">all or nothing first</pl-answer>
+<pl-answer correct="true">all or nothing second</pl-answer>
+<pl-answer correct="true">all or nothing third</pl-answer>
+</pl-order-blocks>
+
+<pl-order-blocks
+answers-name="test-ordered-grading-method-partial-credit"
+grading-method="ordered"
+partial-credit="lcs">
+<pl-answer correct="true">first</pl-answer>
+<pl-answer correct="true">second</pl-answer>
+<pl-answer correct="true">third</pl-answer>
+</pl-order-blocks>
```

Comment:
can you keep the formatting/indentation here consistent with the rest of the file?

Reply:
fixed in [24dbf38](https://github.com/PrairieLearn/PrairieLearn/pull/9292/commits/24dbf38c5226346e5fb55ab48562a10754a69d4a).